package.json

```diff
@@ -15,12 +15,10 @@
     "inherits": "~2.0.3",
     "interpret": "^1.1.0",
     "liftoff": "2.5.0",
-    "lodash": "^4.17.5",
+    "lodash": "^4.17.10",
```

Comment:
Since we don't commit package-lock, this actually doesn't change anything in proper CI, hence this is too small of a change to warrant separate PR, but this is still useful to force locally checked out sources to update to Lodash with better Node 10 support.